src/tests/components/ForecastSummaries.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummaries from "../../components/ForecastSummaries";
+
+describe("ForecastSummaries", () => {
+  const validProps = {
+    forecasts: [
+      {
+        date: 1111111,
+        description: "Stub description 1",
+        icon: 800,
+        temperature: {
+          max: 22,
+          min: 12,
+        },
+      },
+      {
+        date: 2222222,
+        description: "Stub description2",
+        icon: 801,
+        temperature: {
+          max: 24,
+          min: 13,
+        },
+      },
+    ],
+    onForecastSelect: () => {},
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummaries
+        forecasts={validProps.forecasts}
+        onForecastSelect={validProps.onForecastSelect}
+      />
+    );
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Its a good idea (though not mandatory) to include a clear line before the start of your assertions in tests, especially if the arrange part of the test (IE arrange, act, assert) is more than one line long. Just a readability thing more than anything
```
    );

    expect(asFragment()).toMatchSnapshot();
```